boot/bootutil/src/image_ec256.c

```diff
@@ -65,23 +65,20 @@ tinycrypt_import_key(uint8_t *pubkey, uint8_t *cp, uint8_t *end)
         memcmp(param.p, ec_secp256r1_oid, sizeof(ec_secp256r1_oid) - 1)) {
         return -4;
     }
-    if (mbedtls_asn1_get_bitstring_null(&cp, end, &len)) {
+    if (mbedtls_asn1_get_bitstring_null(cp, end, &len)) {
         return -6;
     }
-    if (cp + len != end) {
+    if (*cp + len != end) {
         return -7;
     }
 
     if (len != 2 * NUM_ECC_BYTES + 1) {
         return -8;
     }
-    if (cp[0] != 0x04) {
+    if (*cp[0] != 0x04) {
         return -9;
     }
 
```

Comment:
Perhaps we should add 1 to *cp here before returning, although I guess it doesn't make all that much difference. But it makes a little more sense to do it here, since here is where we are looking for the 0x04.

Unrelated to this change, I suppose we should document somewhere that we only support uncompressed public keys.

Reply:
Will do!